src/components/BooksForm.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createBook } from '../actions/index';
+
+const CATEGORIES = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+const renderCategories = () => (
+  CATEGORIES.map(category => (
+    <option key={`${category}-key`} value={category}>
+      {category}
+    </option>
+  ))
+);
+
+class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: CATEGORIES[0],
+    };
+    this.handleNameChange = this.handleNameChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+    this.handleCategoryChange = this.handleCategoryChange.bind(this);
+  }
+
+  onSubmit(event) {
+    event.preventDefault();
+    const { title, category } = this.state;
+    const { createBook } = this.props;
+    createBook({ title, category });
+  }
+
+  handleNameChange(event) {
+    this.setState({
+      title: event.target.value,
+    });
+  }
+
+  handleCategoryChange(event) {
+    this.setState({
+      category: event.target.value,
+    });
+  }
```

Comment:
**OPTIONAL**: You could have a single handler for both of this https://reactjs.org/docs/forms.html#handling-multiple-inputs
<details>
 <summary>Hint</summary>
 
js```
```javascript
handleChange(event) {
  const { name, value } = event.target;

  this.setState({
    [name]: value,
  })
}
```

</details>